django_project/core/test_views.py

```diff
@@ -7,7 +7,10 @@
 from django.utils.http import urlsafe_base64_encode
```

Comment:
@tinashechiraya can you put this on the test folder, if it doesn't exist then please create it first.